site/formulas/decoding.md

```diff
@@ -1 +1,39 @@
 # Decoding to Extended Coordinates
+
+Given 32 octets of data, `s_bytes`, representing a compressed
+Ristretto point, the implementation must:
+
+1. Check that `s_bytes` is in fact 32 octets.  (In typed languages the
+   implementor should guarantee this through the type system, otherwise
+   there should be an explicit check.)
+2. Decode `s_bytes` into a field element, `s`, then re-encode `s` into
+   another array of 32 octets, `s_bytes_check`.  Check that the two
+   arrays `s_bytes` and `s_bytes_check` are equal in constant time.
```

Comment:
nit: this doesn't need to be done in CT, since it's input validation

Reply:
Removed in 9c06283